tests/phases/design/test_electrical_design.py

```diff
@@ -59,6 +61,78 @@ def test_parameter_sweep(distance_to_landfall, depth, plant_cap, cable):
     assert 1e6 <= o.total_substation_cost <= 1e9
 
 
+def test_calc_substructure_mass_and_cost():
```

Comment:
There should also be a check that the detailed outputs of a default matching a "Fixed" input since we're adding in a new option.

Reply:
Added a test_detailed_design_len()